public/Lab-Files/Lesson02/Assignment/resume.html

```diff
@@ -0,0 +1,92 @@
+<!doctype html>
+ <html lang="en">
+ <head>
+     <meta charset="UTF-8">
+    
+       
+        <title>Priscilla's Resume</title>
+        
+        </head>
+        
+    <body
+   
+        
+        style="background-color: DarkSeaGreen">
```

Comment:
You could write this on line 11 right next to body; there was no need to skip two lines.